ivy/data_classes/array/elementwise.py

```diff
@@ -1815,6 +1843,7 @@ def maximum(
         Examples
         --------
         With :class:`ivy.Array` inputs:
+
```

Comment:
Is this intentional? If so why? I thought you were adding the newline to separate the title from the body or is it to have the line before the codeblock (I seemed to have seen comments on both on your other PR)